client/my-sites/hosting/main.js

```diff
@@ -8,6 +8,7 @@ import { useEffect } from '@wordpress/element';
 import { localize } from 'i18n-calypso';
 import { Fragment, useState, useCallback } from 'react';
 import { connect } from 'react-redux';
+import { useHash } from 'calypso//landing/stepper/hooks/use-hash.ts';
```

Comment:
```suggestion
import { useHash } from 'calypso/landing/stepper/hooks/use-hash.ts';
```

Is this a typo? If so it's not breaking anything but the second forward slash caught my eye.